extend.php

```diff
@@ -27,10 +30,37 @@
         ->js(__DIR__ . '/js/dist/admin.js')
         ->css(__DIR__ . '/less/login2seeplus-settings.less'),
 
-    new Extend\Locales(__DIR__ . '/locale'),
+    (new Extend\Locales(__DIR__ . '/locale')),
 
-    function (Dispatcher $events) {
-        $events->listen(Serializing::class, LoadSettingsFromDatabase::class);
-    }
+    (new Extend\Settings())
+        ->serializeToForum('jslirola.login2seeplus.post', 'jslirola.login2seeplus.post', function ($val) {
+            return $val;
+        })
+        ->serializeToForum('jslirola.login2seeplus.link', 'jslirola.login2seeplus.link', function ($val) {
+            return $val;
+        })
+        ->serializeToForum('jslirola.login2seeplus.image', 'jslirola.login2seeplus.image', function ($val) {
+            return $val;
+        })
+        ->serializeToForum('jslirola.login2seeplus.php', 'jslirola.login2seeplus.php', function ($val) {
+            return $val;
+        })
+        ->serializeToForum('jslirola.login2seeplus.code', 'jslirola.login2seeplus.code', function ($val) {
+            return $val;
+        }),
```

Comment:
You don't need 
```
function ($val) {
    }	            return $val;
```

you can just leave that 3rd argument blank

Reply:
Done!  :)